Detectors/TRD/workflow/src/TRDGlobalTrackingSpec.cxx

```diff
@@ -197,25 +197,18 @@ void TRDGlobalTracking::run(ProcessingContext& pc)
   LOGF(INFO, "%i tracks are loaded into the TRD tracker. Out of those %i ITS-TPC tracks and %i TPC tracks", nTracksLoadedITSTPC + nTracksLoadedTPC, nTracksLoadedITSTPC, nTracksLoadedTPC);
 
   // load the TRD tracklets
-  for (int iTrklt = 0; iTrklt < nTracklets; ++iTrklt) {
-    auto trklt = trackletsTRD[iTrklt];
-    GPUTRDTrackletWord trkltLoad(trklt.getTrackletWord());
-    if (mTracker->LoadTracklet(trkltLoad) > 0) {
-      LOG(WARNING) << "Could not load tracklet " << iTrklt;
-    }
-    if (mUseTrackletTransform) {
+  if (mUseTrackletTransform) {
+    // TODO add space point to GPU Data Types and load them all in one go same as Tracklet64s
+    for (int iTrklt = 0; iTrklt < nTracklets; ++iTrklt) {
       const CalibratedTracklet cTrklt = (cTrkltsPtr->data())[iTrklt];
       mTracker->SetInternalSpacePoint(iTrklt, cTrklt.getX(), cTrklt.getY(), cTrklt.getZ(), cTrklt.getDy());
     }
   }
 
-  mTracker->SetTriggerRecordTimes(&(trdTriggerTimes[0]));
-  mTracker->SetTriggerRecordIndices(&(trdTriggerIndices[0]));
-  mTracker->SetNCollisions(nCollisions);
   //mTracker->DumpTracks();
   mTracker->ResetImpactAngleHistograms();
   mTracker->DoTracking(mChainTracking);
-  //mTracker->DumpTracks();
+  mTracker->DumpTracks();
```

Comment:
leftover debug code, should not be part of the commit

Reply:
Done